internal/configs/parsing_helpers.go

```diff
@@ -225,6 +225,19 @@ func ParseSize(s string) (string, error) {
 	return "", errors.New("Invalid size string")
 }
 
+// https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers
+var proxyBuffersRegexp = regexp.MustCompile(`^\d+ \d+[kKmM]?$`)
+
+// ParseProxyBuffersSpec ensures that the string value is a valid proxy buffer spec
+func ParseProxyBuffersSpec(s string) (string, error) {
```

Comment:
would it help to have a unit test for this func?

Reply:
Thanks, I'd forgotten, added one now.